guides/common/modules/con_content-views-in-projectname.adoc

```diff
@@ -0,0 +1,46 @@
+[id="Content-Views-in-{ProjectNameID}_{context}"]
+= Content Views in {ProjectName}
+
+A _content view_ defines a deliberately curated subset of content that your hosts can access.
+Each content view creates a set of repositories across each environment.
+The {ProjectServer} stores and manages these repositories.
+You can use content views to define which software versions a particular environment or {SmartProxyServer} uses.
+For example:
+
+* You can create a content view with older package versions for a Production environment and another content view with newer package versions for a Development environment.
+* You can create a content view with a package repository required by an operating system and another content view with a package repository required by an application.
+
+You can also combine content views to create a _composite content view_ for ease of management.
+
+.Default Organization View
+A _Default Organization View_ is an application-controlled content view for all content that is synchronized to {Project}.
+Default Organization View useful if you want to register a host to {Project} and access content using a subscription, without manipulating content views and lifecycle environments.
+
+.Promoting a content view across environments
+When you promote a content view from one environment to the next environment in the application life cycle, the respective repository on {ProjectServer} updates and publishes the packages.
+
+In the following example, the repositories for Testing and Production contain the `_example_software_-1.0-0.{client-pkg-arch}.{client-pkg-ext}` package:
```

Comment:
As per the RH SSG, user-replaced values should be italicized and in angled brackets:

```suggestion
In the following example, the repositories for Testing and Production contain the `_<example_software>_-1.0-0.{client-pkg-arch}.{client-pkg-ext}` package:
```

Reply:
Apply below as well.